### ISSUE TITLE
Cannot use because Color32 is not in epaint::color::Color32

### ISSUE THREAD

**Issue by matthiasbeyer** [setzer22/egui_node_graph#96]
This crate expects `epaint::color::Color32` as return value from `DataTypeTrait::data_type_color()`, but `Color32` is at `epaint::Color32`.

Can you please publish a bugfix release for fixing this?

**Comment by matthiasbeyer** [comment]
Ah, presumably that's because the 0.4.0 release depends on egui 0.19, ... so I guess a new release for the new egui version would be nice! :smile:

**Comment by setzer22** [comment]
Yes! We're planning on a new release once a few more things are merged. In the meantime, you can depend on the main git branch. We try to always keep it in a working state (still, it's good to pin a specific commit just in case).

**Comment by matthiasbeyer** [comment]
That's what I'll do! Thanks!

**Comment by JohnDowson** [comment]
> Yes! We're planning on a new release once a few more things are merged. In the meantime, you can depend on the main git branch. We try to always keep it in a working state (still, it's good to pin a specific commit just in case).

It's been awhile, any plans on cutting a new release? Docs seem too be drastically better on master as well as fresh egui and a bunch of features, judging by commit log.
